docs/docs/guides/database-queries.mdx

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_position: 3
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+# Helpful database queries for advanced debugging of Immich
+
+## Prerequisites
+
+In order to execute the queries, you must first connect to the container and get the postgres shell.
+
+### Docker Compose
+
+If you are using docker compose, simply run `docker exec -it immich_postgres psql immich DB_USERNAME` while `DB_USERNAME`
+should be replaced by the according value [set in your `.env` file](https://docs.immich.app/docs/install/environment-variables#database).
+
+### GUI (Portainer, Unraid, ...)
+
+When using a GUI, it will most likely provide the ability to create a shell within the container.
+In that shell, simply run `psql immich DB_USERNAME` while replacing your `DB_USERNAME` like explained above.
+
+### Kubernetes
+
+If you're using Kubernetes in your home lab you are crazy and know how to run a command anyway.
```

Comment:
Nit: I don't know if that should call people crazy. ;)

Reply:
Thanks for the review!
That's why it's still a draft :)